### PR TITLE
2330 added bash script, changed docker-compose file, added info in RE…

### PR DESCRIPTION
**Pull request by mikhail-balabaev**
Work item [AB#2330](https://deloittedigitalat.visualstudio.com/38398c5c-11b2-4e64-98b5-5af5e65d6e82/_workitems/edit/2330)
- added bash script, 
- changed docker-compose file, 
- added info in README

### CODE REVIEW

File: data-service/README.md [hunk 1]
```diff
@@ -74,6 +74,18 @@ docker build -t data-service .
...
+
+```
+./docker-build.sh
+```
+
+and from core-banking/devops/docker/ run
+
+```
+docker-compose up -d
+```
```
Comment (theqajedi) [inline data-service/README.md:87]: can you also add this in the main readme?
its missing there.
basically how to start all the services on dev.

Comment (aberke220) [reply data-service/README.md:87]: we need a bit more explanation for that :) since we also have to build all containers.

Comment (mikhail-balabaev) [reply data-service/README.md]: Added instruction in root directory README how to run current services and db locally. Please check.

File: README.md [hunk 2]
```diff
@@ -46,4 +46,39 @@ Configuration guide is available at https://github.com/DeloitteDigitalAT/core-ba
...
 
-- TBD
\ No newline at end of file
+- TBD
+
+## How to run services and db locally. Requires JDK 11+ and Docker
+1) Build containers
+```
+cd data-access-services
+mvn install
```
Comment (aberke220) [inline README.md:55]: `mvn install` is not needed anymore as it is part of the `./docker-build.sh`

File: README.md [hunk 3]
```diff
@@ -46,4 +46,39 @@ Configuration guide is available at https://github.com/DeloitteDigitalAT/core-ba
...
+```
+docker ps
+```
+The following images should be added:
+- core-banking/data-service
+- core-banking/accounts-service
+- core-banking/transactions-service
+- core-banking/credit-application-service
+
+4) Check endpoints
```
Comment (aberke220) [inline README.md:80]: Thanks for adding these!

File: README.md [hunk 4]
```diff
@@ -46,4 +46,39 @@ Configuration guide is available at https://github.com/DeloitteDigitalAT/core-ba
...
+
+cd ../domain-service/credit-application/
+./docker-build.sh
+
+cd ../data-service
+./docker-build.sh
+```
+
+2) Run docker compose
+From root directory:
```
Comment (theqajedi) [inline README.md:66]: its moved to `devops/docker` dir.

File: README.md (hunk 2 above)
Comment (mikhail-balabaev) [reply README.md]: good to know, removed line 'mvn install'

File: README.md (hunk 4 above)
Comment (mikhail-balabaev) [reply README.md:65]: changed instruction. Added 'cd devops/docker/'

**Review by theqajedi — APPROVED** [review]

**Pull request merged**